README.md

```diff
@@ -41,6 +41,15 @@ Download the appropriate [release](https://github.com/ethanmoffat/EndlessClient/
 
 After installing, clone (or fork+clone) this repository locally and open the solution in your IDE of choice for your platform.
 
+### Building on Mac
+
+1. Download and install the [.NET 6.0 SDK (x64)](https://dotnet.microsoft.com/en-us/download/dotnet/thank-you/sdk-6.0.403-macos-x64-installer).
+2. Link the binary to so it's in path `sudo ln -s /usr/local/share/dotnet/x64/dotnet /usr/local/bin/dotnet`
```

Comment:
I didn't have this issue when I ran on my mac VM. Maybe this error only shows up in certain versions? I was using macOS Monterey.

Reply:
Are you referring to needing to link to the binary? That might have just been an issue for me.

Reply:
Yeah - my guess is that I restarted the terminal or the VM and that reloaded the environment variables so I don't think this step is necessary

Reply:
I'm going to not care about this for now so we can get the PR merged